ui/widgets/menu.js

```diff
@@ -92,7 +92,9 @@ return $.widget( "ui.menu", {
 					// Open submenu on click
 					if ( target.has( ".ui-menu" ).length ) {
 						this.expand( event );
-					} else if ( !this.element.is( ":focus" ) && $( $.ui.safeActiveElement( this.document[ 0 ] ) ).closest( ".ui-menu" ).length ) {
+					} else if ( !this.element.is( ":focus" ) &&
+							$( $.ui.safeActiveElement( this.document[ 0 ] ) )
+							.closest( ".ui-menu" ).length ) {
```

Comment:
This line needs to be indented one more level, but that gets really crazy. Let's store the active element in a var to reduce the save.